ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -129,6 +129,13 @@ public final class AliasTokens: NSObject {
         case brandStroke1
         case brandStroke1Pressed
         case brandStroke1Selected
+        case brandStrokeTint
+
+        // Shared colors - Stroke
+        case dangerStroke1
+        case successStroke1
+        case warningStroke1
+        case severeStroke1
```

Comment:
I think the tokens might be mostly sorted alphabetically by category, so maybe these should go after `// Shared colors - Presence`?